pkgs/by-name/iw/iwd/package.nix

```
@@ -13,6 +14,23 @@
   gitUpdater,
 }:
 
+let
+  readline-patch = fetchpatch {
```

```suggestion
  # fix segfault in iwctl with readline-8.3
  # https://lists.gnu.org/archive/html/bug-readline/2025-07/msg00007.html
  readline-patch = fetchpatch {
```

Add some context and link https://lists.gnu.org/archive/html/bug-readline/2025-07/msg00007.html.